### DISCUSSION TITLE
Smartlook session management

### DISCUSSION THREAD

**Discussion by rnjt-sp6** [smartlook/smartlook-mobile-issue-tracker#113] [Q&A]
Hey Guys, I need to understand the possibility of maintaining the same session throughout the day even if user close the app for a bit long interval. Is that possible?

**Comment by pavel-kroh-smartlook** [comment]
Hi @rnjt-sp6 , 

it is currently not possible. 

"Session" is designed as "an app run". All sessions recorded by the same instance of the app (i.e., on the same device) are grouped under the same user, however, so you can find them there.

What is the particular use case?

Pavel

**Reply by rnjt-sp6** [reply]
Hey @pavel-kroh-smartlook , Our app is used by delivery agents where they will need to pickup and drop parcels. Currently it takes almost 4-5 session for a single delivery. Our need is to maintain the full flow of the day in a session.

**Reply by pavel-kroh-smartlook** [reply]
Hi @rnjt-sp6 , I see, this makes sense. I'll share the question/feature request with my colleagues to discuss its feasibility.

**Reply by rnjt-sp6** [reply]
Thanks @pavel-kroh-smartlook for the quick response, Yeah it will be so useful if this feature available
